osu.Game/Screens/Select/BeatmapDeleteDialog.cs

```diff
@@ -14,7 +14,7 @@ public partial class BeatmapDeleteDialog : DangerousActionDialog
         public BeatmapDeleteDialog(BeatmapSetInfo beatmapSet)
         {
             this.beatmapSet = beatmapSet;
-            BodyText = $@"{beatmapSet.Metadata.Artist} - {beatmapSet.Metadata.Title}";
+            BodyText = $@"You are PERMANENTLY DELETING: {beatmapSet.Metadata.Artist} - {beatmapSet.Metadata.Title}";
```

Comment:
It's not permanent until you close the game

Reply:
that is understandable, but it would be more userfriendly to mention this, regardless of the time of deletion. Could you suggest what you had in mind to replace this with?

Reply:
Just removing the word "permanent" would be fine by me

Reply:
done with the latest commit